app/database/operator/server_data_operator/handlers/group.ts

```diff
@@ -106,6 +108,66 @@ const GroupHandler = (superclass: any) => class extends superclass implements Gr
 
         return records;
     };
+
+    /**
+     * handleGroupTeamsForTeam: Handler responsible for the Create/Update operations occurring on the GroupTeam table from the 'Server' schema
+     * @param {string} teamId
+     * @param {HandleGroupTeamsForTeamArgs} groupTeamsArgs
+     * @param {GroupTeam[]} groupTeamsArgs.groupTeams
+     * @param {boolean} groupTeamsArgs.prepareRecordsOnly
+     * @throws DataOperatorException
+     * @returns {Promise<GroupTeamModel[]>}
```

Comment:
```suggestion
     * @param {HandleGroupTeamsForTeamArgs} groupTeamsArgs
     * @param {GroupTeam[]} groupTeamsArgs.teamId
     * @param {GroupTeam[]} groupTeamsArgs.groups
     * @param {boolean} groupTeamsArgs.prepareRecordsOnly
     * @returns {Promise<GroupTeamModel[]>}
```

Reply:
Good catch, have fixed for the other methods too 🙌